src/core/selection/select.ts

```diff
@@ -406,7 +406,7 @@ export class Select implements ISelect {
 			preventScroll: true
 		}
 	): boolean {
-		if (!this.isFocused()) {
+		if (!this.isFocused() && !options.preventScroll) {
```

Comment:
In this case, the method does nothing at all, it does not even set the focus.

Reply:
Ah, it worked though 😅  

I made modifications based on your feedback. Thank you